src/test/java/org/gridsuite/modification/server/Impacts/TestImpactUtils.java

```diff
@@ -56,16 +58,17 @@ private static void testEmptyImpacts(ApplicationStatus globalApplicationStatusEx
         assertThat(networkModificationResult).recursivelyEquals(resultExpected);
     }
 
-    public static void testElementImpacts(ObjectMapper mapper, String resultAsString, int nbImpacts, Set<String> substationIds) throws JsonProcessingException {
+    public static void testElementImpacts(ObjectMapper mapper, String resultAsString, int nbImpacts, Set<IdentifiableType> collectionImpactElementTypes, Set<String> substationIds) throws JsonProcessingException {
         Optional<NetworkModificationResult> networkModificationResult = mapper.readValue(resultAsString, new TypeReference<>() { });
         assertTrue(networkModificationResult.isPresent());
 
         assertEquals(ApplicationStatus.ALL_OK, networkModificationResult.get().getApplicationStatus());
         assertEquals(new TreeSet<>(substationIds), networkModificationResult.get().getImpactedSubstationsIds());
         assertEquals(nbImpacts, networkModificationResult.get().getNetworkImpacts().size());
+        assertTrue(networkModificationResult.get().getNetworkImpacts().containsAll(collectionImpactElementTypes.stream().map(t -> createCollectionElementImpact(t)).toList()));
```

Comment:
```suggestion
assertThat(networkModificationResult.get().getNetworkImpacts()).containsExactlyInAnyOrderElementsOf(collectionImpactElementTypes.stream().map(TestImpactUtils::createCollectionElementImpact).toList());
```

Reply:
kept `containsAll` because it could have `CollectionImpacts` and `SimpleElementImpacts` as well